osquery/database/plugins/rocksdb.cpp

```diff
@@ -228,9 +228,9 @@ rocksdb::DB* RocksDBDatabasePlugin::getDB() const {
 rocksdb::ColumnFamilyHandle* RocksDBDatabasePlugin::getHandleForColumnFamily(
     const std::string& cf) const {
   try {
-    for (size_t i = 0; i < kDomains.size(); i++) {
-      if (kDomains[i] == cf) {
-        return handles_[i];
+    for (auto handle : handles_) {
```

Comment:
`const auto&` ?

Reply:
Here `handle` should be a `rocksdb::ColumnFamilyHandle*`, does it make sense to use a const ref in this case?

Reply:
Yeah good point. You could still make this `const auto` though yeah?